.github/workflows/jit.yml

```diff
@@ -58,7 +58,7 @@ jobs:
           - aarch64-pc-windows-msvc/msvc
           - x86_64-apple-darwin/clang
           - aarch64-apple-darwin/clang
-          - x86_64-unknown-linux-gnu/gcc
+          - x86_64-pc-linux-gnu/gcc
```

Comment:
Turns out that this was actually running on `pc` the whole time and the superficial name was incorrect.